content/applications/inventory_and_mrp/inventory/management/products/reordering_rules.rst

```diff
@@ -0,0 +1,126 @@
+create a new one by clicking :guilabel:`Create`.
+
+First, on the :guilabel:`General Information` tab, make sure that the :guilabel:`Product Type` is
+set to :guilabel:`Storable Product`. This is necessary because Odoo only tracks stock quantities for
+storable products, and this number is used to trigger reordering rules.
+
+.. image:: reordering_rules/product-type.png
+   :align: center
+   :alt: Set the Product Type as Storable.
+
+Next, click on the :guilabel:`Inventory` tab and select one or more routes from the
+:guilabel:`Routes` section. Doing so tells Odoo which route to use to replenish the product.
+
+.. image:: reordering_rules/select-routes.png
+   :align: center
+   :alt: Select one or more routes on the Inventory tab.
+
+If the product is reordered using the :guilabel:`Buy` route, confirm that the :guilabel:`Can be
+Purchased` checkbox is enabled under the product name. This makes the :guilabel:`Purchase` tab
+appear. Click on the :guilabel:`Purchase` tab, and specify at least one vendor, and the price that
+they sell the product for, so that Odoo knows which company the product should be purchased from.
+
+.. image:: reordering_rules/specify-vendor.png
+   :align: center
+   :alt: Specify a vendor and price on the Purchase tab.
+
+If the product is replenished using the :guilabel:`Manufacture` route, it needs to have at least one
+Bill of Materials (BoM) associated with it. This is necessary because Odoo only creates
+Manufacturing Orders for products with a :abbr:`BoM (Bill of Materials)`.
+
+If a :abbr:`BoM (Bill of Materials)` does not already exist for the product, select the
+:guilabel:`Bill of Materials` smart button at the top of the product form, then click
+:guilabel:`Create` to configure a new :abbr:`BoM (Bill of Materials)`.
+
+.. image:: reordering_rules/BoM-smart-button.png
+   :align: center
+   :alt: The Bill of Materials smart button on a product form.
+
+Create new reordering rules
+===========================
+
+To create a new reordering rule, navigate to :menuselection:`Inventory --> Configuration -->
+Reordering Rules`, then click :guilabel:`Create`, and fill out the new line as follows:
+
+- :guilabel:`Product`: The product that is replenished by the rule.
+- :guilabel:`Location`: The location where the product is stored.
+- :guilabel:`Min Quantity`: The minimum quantity that can be forecasted without the rule being
+  triggered. When forecasted stock falls below this number, a replenishment order for the product is
+  created.
+- :guilabel:`Max Quantity`: The maximum quantity that stock is replenished up to.
+- :guilabel:`Multiple Quantity`: Specify if the product should be replenished in batches of a
+  certain quantity (e.g., a product could be replenished in batches of 20).
+- :guilabel:`UoM`: The unit of measure used for reordering the product. This value can simply be
+  `Units` or a specific unit of measurement for weight, length, etc.
+
+.. image:: reordering_rules/reordering-rule-form.png
```

Comment:
This image is barely legible, can you please re-compress from the original or retake with less width, I dont feel it is necessary to show the entire screen here. Thank you 🙏